skimage/color/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""Utilities for color conversion, color labeling, and color difference
```

Comment:
I think that "color space conversion" would be clearer (and more accurate) than "color conversion" -- feel free to submit a follow-up PR @harshithakolipaka 

/cc @lagru